utils/utils.py

```diff
@@ -707,5 +707,5 @@ def suffix_pattern(suffixes):
     return re.compile("(?:%s)$" % center)
 
 
-def flatten(l):
-    return list(chain.from_iterable(l))
+def flatten(iterable):
```

Comment:
This is actually outside the scope of my scanner, but that `l` var make circle unhappy.

Reply:
Thanks for fixing this one! Just showed up in my linter too.